checkov/terraform/checks/graph_checks/aws/S3BucketGlobalViewACLDisabled.yaml

```diff
@@ -0,0 +1,17 @@
+metadata:
+  name: "Ensure AWS S3 bucket has global view ACL permissions disabled"
```

Comment:
I think this check is not needed, because we have CKV_AWS_20 and CKV_AWS_57

Reply:
Noted @gruebel . I've deleted that policy.